content/docs/command-reference/push.md

```diff
@@ -71,9 +71,9 @@ in the cache (compared to the default remote.) It can be used to see what files
   or project. Note that both options can be combined, for example using the
   `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git  
-  commits as well as the workspace. Useful for uploading all the data used in
-  the entire existing commit history of the project.
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
+  as well as the workspace. Useful for downloading all the data used in the
```

Comment:
same here - why downloading?

Reply:
Fixed in 857919e6 as well. PTAL (I changed a few other refs for correctness and consistency).